### ISSUE TITLE
Sometimes a project shows multiple icons for one user (multiple copies) - BUG

### ISSUE THREAD

**Issue by DS-Matt** [mirumee/overv.io-beta#530]
As you can see from the screengrab - the project shows multiple images of one of our Github users (who is on the repos of this project - but hasn't been invited to work in Overv.io - so she shouldn't be showing at all).

![multiple copies of users on a project](https://cloud.githubusercontent.com/assets/1852169/12760332/bf0c31bc-c99a-11e5-9598-1ec412ff11b4.png)

**Comment by DS-Matt** [comment]
After testing, it appears the users get added when clicking on the "Create new issue" button!

**Comment by mirekm** [comment]
Thanks for the feedback but we still can't really reproduce this issue. Do you mean the (+) button in the bottom right corner?

**Comment by DS-Matt** [comment]
That is correct, when I click the single plus button - I'll make a video recording and post it.

Thanks!

**Comment by DS-Matt** [comment]
http://sendvid.com/hxea6rzt

**Comment by DS-Matt** [comment]
@mirekm -

When using Overv.io today I noticed this doesn't seem to happen anymore, has this issue been addressed/fixed by your dev team?  If so I'll close this issue, I just don't want to close this issue if the bug hasn't been addressed yet and it just seems to have temporarily gone away...
